auth_oauth_enforced/readme/CONTEXT.md

```diff
@@ -0,0 +1 @@
+To streamline security practices, many companies are adopting an OAuth-only login policy, reducing risks associated with managing multiple passwords. However, in B2B scenarios—such as when third-party service providers like accounting firms require access to Odoo—enforcing OAuth exclusively can lead to operational inefficiencies, as third parties may need to create and manage dedicated accounts for this purpose. To overcome this challenge, implementing domain-specific OAuth enforcement offers a more flexible and targeted solution.
```

Comment:
Make it more concise as it sounds a bit too verbose (the last sentence sounds very ChatGPTish and we don't want that), and apply the 88-char max length for better readability, although it's not required by the linter.